initrd/bin/kexec-boot

```diff
@@ -155,4 +155,8 @@ if [ -x /bin/io386 -a "$CONFIG_FINALIZE_PLATFORM_LOCKING_PRESKYLAKE" = "y" ]; th
 fi
 
 echo "Starting the new kernel"
+if [ "$CONFIG_DEBUG_OUTPUT" = "y" ];then
+        #Repeat kexec command that will be executed since in debug
+        DEBUG "kexeccmd= $kexeccmd"
```

Comment:
@tlaurion I was just adding this too :joy: Would you mind adding:
```
	read -n 1 -p "[DEBUG] Continue booting? [Y/n]: " debug_boot_confirm
	if [ "$debug_boot_confirm^^" = Y ]; then
		# abort
		die "Boot aborted"
	fi
```

So we could abort the boot in debug for inspection.  Should also move it above the `tpmr kexec_finalize` and preskylake lock then IMO

Reply:
done